src/less/listbox-button/base/listbox-button.less

```diff
@@ -50,6 +49,17 @@ svg.listbox-button__status {
 }
 
 // background icon
+.listbox-button__status {
```

Comment:
If I remember correctly, these elements are flex children and so these first and last child margin rules were replaced by me with a simple 'margin-left: auto'. If they are not flex children (perhaps they only are in certain dropdowns), then ignore my comment.

Reply:
Here: https://github.com/eBay/skin/commit/28c696b1479f82da9163f26364e6377d512defb0#diff-4babd5c8f7e99f21b79c4125ef2e5c3bR65

Reply:
@ianmcburnie The thing is when I don't remove these styles it looks like this: 
![image](https://user-images.githubusercontent.com/1755269/62494491-82d5dd80-b788-11e9-8abd-72cb192216df.png)
And the styles are applied as so, 
![image](https://user-images.githubusercontent.com/1755269/62494532-9d0fbb80-b788-11e9-90d4-9190e64051d9.png)

If that looks fine, then I'm fine with it too.

Reply:
Nope. We need the 8px margin (or padding) on the left of the check mark. Something to make sure there's always space.

Reply:
@agliga Gotcha. So long as this doesn't revert menu and combobox, and is listbox specific, then I'm fine with it. At some point I'll take a look and see why they are all different.